webtranslate/users/github.py

```diff
@@ -0,0 +1,134 @@
+"""
+User authentication and authorization via github teams.
+
+"""
+import configparser, os, json, requests, requests_oauthlib
```

Comment:
`requests_oauthlib` is introduced here, but I don't see a `requirements.txt` or `setup.py` or `README.md` change that tells the user about it. How do you know which dependencies to install for eints?

Reply:
Depending on the settings in config.xml, it either needs requests-oauthlib, postgres or other libs.
Most of these backends are now deprecated, I guess.